tox.ini

```diff
@@ -1,22 +1,21 @@
 [tox]
 skip_missing_interpreters = True
 skipsdist = True
-usedevelop = True
 
 envlist =
-    py{27,33,34,35}-dj{18,19,110}-wt16
+    py{27,33,34,35}-dj{18,19,110}-wt{18,19,110,111}
```

Comment:
Can you add dj111 to the envlist because it is defined below but not being used to test.